model/src/order.rs

```diff
@@ -258,21 +258,70 @@ impl OrderCreation {
     }
 }
 
-// /// An order cancellation as provided to the orderbook by the frontend.
-// #[serde_as]
-// #[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize, Hash)]
-// #[serde(rename_all = "camelCase")]
-// pub struct OrderCancellation {
-//     pub uid: OrderUid,
-//     pub signature: Signature,
-// }
+/// An order cancellation as provided to the orderbook by the frontend.
+#[serde_as]
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize, Hash)]
+pub struct OrderCancellation {
+    // TODO - does it make sense to include the orderUid here?
+    pub order_uid: OrderUid,
+    pub signature: Signature,
+}
+
+impl OrderCancellation {
```

Comment:
A test case would be nice so that the frontend can verify how the signing logic needs to look like

Reply:
This is also part of the todo.